### DISCUSSION TITLE
WG - Moving Quarkus to the CommonHaus Foundation

### DISCUSSION THREAD

**Discussion by maxandersen** [quarkusio/quarkus#43013] [Design Discussions]
We started the move of Quarkus to a foundation [earlier this year](https://quarkus.io/blog/quarkus-in-a-foundation/) and recently [set the direction](https://quarkus.io/blog/quarkus-moving-to-commonhaus/) towards [CommonHaus](https://www.commonhaus.org/) and during the summer break the CommonHaus council [approved our request](https://github.com/commonhaus/foundation/pull/183) to join.

Thus, now the real work starts, and it's just fitting we set up a working group for the effort getting Quarkus to CommonHaus foundation.

# Goal

Two parts 

- setup Quarkus to have transparent and open governance
- Go through the few but important requirements for a CommonHaus project.   

# Initial work items/questions:

Current known list, but not limited to:

- identify design communication channels (i.e. #41973)
- which repositories / code will move
- impact (if any) on quarkiverse projects  
- how will trademarks work/change
- identify running services and setup/maintain them (registry.quarkus.io, code.quarkus.io etc.)
- add required metadata/files to the various repositories

# Tracking

We will use the working group board to track publicly all the known relevant work and questions.For the few exception cases where, for legal or personal constraints, the work must happen in private, we will post the outcome and results in public places (like a GitHub discussion of a GitHub issue tracked on the working group board).

# When will this working group be done?

When Quarkus has an active working governance model in place and all major work items around setting up Quarkus at CommonHaus are completed - after that, its expected things will just be iteratively improved, and the dedicated working group will not be needed (others might start to continue more specific efforts).

The majority of the work must be done before the end of December 2024. The latest deadline for CommonHaus is April 2025, when the bootstrap period of CommonHaus ends.

# How can I join?

Leave a comment on this, and you'll ensure you get notified of updates and upcoming movements.

**Comment by emmanuelbernard** [comment]
subscribe

**Comment by cescoffier** [comment]
+1 for having a dedicated working group.
